src/components/alerts/alerts.svelte

```diff
@@ -1,34 +1,45 @@
 <script>
-	import { formatTimestamp } from '$lib/formatters';
+	import { translate, formatTimestamp } from '$lib/formatters';
+	import { getLocale } from '$lib/paraglide/runtime.js';
+	import * as t from '$lib/paraglide/messages.js';
 
 	let { situations = [] } = $props();
 
-	const situation = situations[0];
-	const summary = situation?.summary;
-	const activeWindow = situation?.activeWindows?.[0];
+	let title = $state(situations[0]?.summary?.value?.trim?.() ?? '');
+	let translatedTitle = $state(title);
 
-	const title = summary?.value?.trim?.() ?? '';
+	$effect(() => {
+		if (getLocale() !== 'en') {
+			translate(title, getLocale()).then((result) => {
+				translatedTitle = result;
+			});
+		}
+	});
+
+	const activeWindow = situations[0]?.activeWindows?.[0];
 	const dateStart = formatTimestamp(activeWindow?.from);
 	const dateEnd = formatTimestamp(activeWindow?.to);
+
+	const alignment = getLocale() !== 'ar' ? `` : `right-alignment leading-12`;
```

Comment:
let's get rid of all of these explicit `getLocale()` checks and try to make the alignment entirely implicit via the `dir` attribute.